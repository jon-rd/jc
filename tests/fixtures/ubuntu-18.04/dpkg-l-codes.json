[{"codes":"ii","name":"accountsservice","version":"0.6.45-1ubuntu1.3","architecture":"amd64","description":"query and manipulate user account information","desired":"install","status":"installed"},{"codes":"rc","name":"acl","version":"2.2.52-3build1","architecture":"amd64","description":"Access control list utilities","desired":"remove","status":"config-files"},{"codes":"uWR","name":"acpi","version":"1.7-1.1","architecture":"amd64","description":"displays information on ACPI devices","desired":"unknown","status":"trigger await","error":"reinstall required"},{"codes":"rh","name":"acpid","version":"1:2.0.28-1ubuntu1","architecture":"amd64","description":"Advanced Configuration and Power Interface event daemon","desired":"remove","status":"half installed"},{"codes":"pn","name":"adduser","version":"3.116ubuntu1","architecture":"all","description":"add and remove users and groups","desired":"purge","status":"not installed"},{"codes":"ii","name":"amd64-microcode","version":"3.20191021.1+really3","architecture":"amd64","description":"Processor microcode firmware for AMD CPUs","desired":"install","status":"installed"},{"codes":"ii","name":"apparmor","version":"2.12-4ubuntu5.1","architecture":"amd64","description":"user-space parser utility for AppArmor","desired":"install","status":"installed"},{"codes":"ii","name":"apport","version":"2.20.9-0ubuntu7.23","architecture":"all","description":"automatically generate crash reports for debugging","desired":"install","status":"installed"},{"codes":"ii","name":"apport-symptoms","version":"0.20","architecture":"all","description":"symptom scripts for apport","desired":"install","status":"installed"},{"codes":"rc","name":"linux-image-4.15.0-136-generic","version":"4.15.0-136.140","architecture":"amd64","description":"Signed kernel image generic","desired":"remove","status":"config-files"},{"codes":"ii","name":"linux-image-4.15.0-137-generic","version":"4.15.0-137.141","architecture":"amd64","description":"Signed kernel image generic","desired":"install","status":"installed"},{"codes":"ii","name":"linux-image-4.15.0-139-generic","version":"4.15.0-139.143","architecture":"amd64","description":"Signed kernel image generic","desired":"install","status":"installed"},{"codes":"ii","name":"linux-image-4.15.0-140-generic","version":"4.15.0-140.144","architecture":"amd64","description":"Signed kernel image generic","desired":"install","status":"installed"},{"codes":"rc","name":"linux-image-4.15.0-55-generic","version":"4.15.0-55.60","architecture":"amd64","description":"Signed kernel image generic","desired":"remove","status":"config-files"},{"codes":"rc","name":"linux-image-4.15.0-58-generic","version":"4.15.0-58.64","architecture":"amd64","description":"Signed kernel image generic","desired":"remove","status":"config-files"},{"codes":"rc","name":"linux-image-4.15.0-65-generic","version":"4.15.0-65.74","architecture":"amd64","description":"Signed kernel image generic","desired":"remove","status":"config-files"}]
