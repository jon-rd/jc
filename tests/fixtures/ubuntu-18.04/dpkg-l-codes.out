Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                            Version              Architecture         Description
+++-===============================-====================-====================-===================================================================
ii  accountsservice                 0.6.45-1ubuntu1.3    amd64                query and manipulate user account information
rc  acl                             2.2.52-3build1       amd64                Access control list utilities
uWR acpi                            1.7-1.1              amd64                displays information on ACPI devices
rh  acpid                           1:2.0.28-1ubuntu1    amd64                Advanced Configuration and Power Interface event daemon
pn  adduser                         3.116ubuntu1         all                  add and remove users and groups
ii  amd64-microcode                 3.20191021.1+really3 amd64                Processor microcode firmware for AMD CPUs
ii  apparmor                        2.12-4ubuntu5.1      amd64                user-space parser utility for AppArmor
ii  apport                          2.20.9-0ubuntu7.23   all                  automatically generate crash reports for debugging
ii  apport-symptoms                 0.20                 all                  symptom scripts for apport
rc  linux-image-4.15.0-136-generic  4.15.0-136.140       amd64                Signed kernel image generic
ii  linux-image-4.15.0-137-generic  4.15.0-137.141       amd64                Signed kernel image generic
ii  linux-image-4.15.0-139-generic  4.15.0-139.143       amd64                Signed kernel image generic
ii  linux-image-4.15.0-140-generic  4.15.0-140.144       amd64                Signed kernel image generic
rc  linux-image-4.15.0-55-generic   4.15.0-55.60         amd64                Signed kernel image generic
rc  linux-image-4.15.0-58-generic   4.15.0-58.64         amd64                Signed kernel image generic
rc  linux-image-4.15.0-65-generic   4.15.0-65.74         amd64                Signed kernel image generic
